packages/a/aws-sdk-cpp/xmake.lua

```diff
@@ -5,13 +5,9 @@ package("aws-sdk-cpp")
     add_urls("https://github.com/aws/aws-sdk-cpp.git")
     add_versions("1.9.362", "e9372218a2c8fab756ecaa6e4fefcdb33c3670c1")
 
-    if is_plat("macosx") then
-        add_frameworks("Foundation", "CoreFoundation", "Security", "SystemConfiguration")
```

Comment:
we need also them if http_client is enabled.

Reply:
👌